lib/openxml/pptx/parts/image.rb

```diff
@@ -0,0 +1,32 @@
+module OpenXml
+  module Pptx
+    module Parts
+      class Image < OpenXml::Parts::UnparsedPart
+        attr_reader :path, :content_type, :relationship_type
+
+        def initialize(*args, extension: ".png")
+          super(*args)
+          @path = Pathname.new("ppt/media/image#{extension}")
```

Comment:
❔ I'm probably reading this wrong — it looks like every `.png` will have the same path?? How do you get different paths if you use more than one image of the same type?

Reply:
In the `OpenXml::RelatablePart` module, the path of a part instance gets rewritten with an index if it's an indexed part (like slide, slide layout, slide master, etc.). The other parts fully conform to `RelatablePart`, since they are not only related to, but can themselves relate to others, i.e., they have a `Rels` part. Image doesn't need a `Rels` part, but still wants to be able to be related to, hence the partial conformance to the `RelatablePart` in the form of having `path`, `path=`, `content_type`, and `relationship_type` methods.